hr_expense_tags/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2023 ForgeFlow - Elmer García / Joan Solé
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Hr expense tags",
```

Comment:
How about joining this module and hr_expense_sheet_tags from https://github.com/OCA/hr-expense/pull/186?